dev-lang/rust/files/1.32.0-libressl.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/vendor/openssl-sys/build/main.rs b/vendor/openssl-sys/build/main.rs
+index 2b1691230..69def400e 100644
+--- a/vendor/openssl-sys/build/main.rs
++++ b/vendor/openssl-sys/build/main.rs
+@@ -1,9 +1,9 @@
+ extern crate cc;
++#[cfg(feature = "vendored")]
++extern crate openssl_src;
+ extern crate pkg_config;
+ #[cfg(target_env = "msvc")]
+ extern crate vcpkg;
+-#[cfg(feature = "vendored")]
+-extern crate openssl_src;
+ 
+ use std::collections::HashSet;
+ use std::env;
+@@ -103,20 +103,23 @@ fn main() {
+ 
+ #[cfg(feature = "vendored")]
+ mod imp {
+-	use std::path::PathBuf;
+-	use openssl_src;
+-
+-	pub fn get_openssl(_target: &str) -> (PathBuf, PathBuf) {
+-		let artifacts = openssl_src::Build::new().build();
+-		(artifacts.lib_dir().to_path_buf(), artifacts.include_dir().to_path_buf())
+-	}
++    use openssl_src;
++    use std::path::PathBuf;
++
++    pub fn get_openssl(_target: &str) -> (PathBuf, PathBuf) {
++        let artifacts = openssl_src::Build::new().build();
++        (
++            artifacts.lib_dir().to_path_buf(),
++            artifacts.include_dir().to_path_buf(),
++        )
++    }
+ }
+ 
+ #[cfg(not(feature = "vendored"))]
+ mod imp {
+     use pkg_config;
+-    use std::path::{Path, PathBuf};
+     use std::ffi::OsString;
++    use std::path::{Path, PathBuf};
+     use std::process::{self, Command};
+ 
+     use super::env;
+@@ -500,6 +503,8 @@ See rust-openssl README for more information:
+             (7, _) => ('7', 'x'),
+             (8, 0) => ('8', '0'),
+             (8, 1) => ('8', '1'),
++            (8, _) => ('8', 'x'),
++            (9, 0) => ('9', '0'),
```

Comment:
This is the only functional change. Let's drop all the other movement and just keep this.